assets/src/editor/blocks/landing-page-hero/index.js

```diff
@@ -205,9 +255,15 @@ export const settings = {
 							tagName="h1"
 							value={ title }
 						/>
+						<RichText.Content
+							className="hero__description"
+							multiline="p"
+							tagName="div"
+							value={ description }
+						/>
 						{ buttonLink && (
 							<a
-								className="hero__call-to-action cta-button"
+								className={ `hero__call-to-action cta-button hero__cta-button hero__cta-button--${ ctaButtonStyle }` }
```

Comment:
Here we'll need to be careful not to change the default class name, or we'll break validation of any existing blocks. Can the additional classes only be added if `ctaButtonStyle` is defined?

Reply:
Thanks for the heads up. I'll figure out how to fix it.